src/components/menu-bar/menu-bar.jsx

```diff
@@ -683,8 +683,7 @@ const mapDispatchToProps = dispatch => ({
     onRequestCloseLogin: () => dispatch(closeLoginMenu()),
     onClickNew: canSave => dispatch(requestNewProject(canSave)),
     onClickSave: () => dispatch(saveProject()),
-    onSeeCommunity: () => dispatch(setPlayer(true)),
-    onShare: () => {} // NOTE: implement this
+    onSeeCommunity: () => dispatch(setPlayer(true))
```

Comment:
Need to also remove the onShare prop above in proptypes

Reply:
We agreed that this is ok, we're not completely removing the prop, just leaving it as undefined by default